docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+      - media:/files/media
+    command: >
+      sh -c "python manage.py migrate &&
+      python manage.py wait_for_db &&
+      python manage.py runserver 0.0.0.0:8000"
```

Comment:
The `command` should ensure that the database is ready before running migrations and starting the server. Consider using a wait-for-it script or similar solution to ensure the database is ready.